style.css

```diff
@@ -1,122 +1,137 @@
-*{
-    font-family: 'Poppins';
-}
-body{
-    background-color: #f3f3f3;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    overflow: hidden;
-}
-
-.top{
-    background-image: url(img/header-background-mobile.png);
-    background-repeat: no-repeat;
-    width: 375px;
-    height: 718px;
-    background-color: white;
-    z-index: inherit;
-}
-nav{
-    position: absolute;
-    top: 50px;
-    left: 24px;
-    width: 100%;
-    overflow: hidden;
-}
-.topnav{
-    display: flex;
-    justify-content: space-between;
-    width: 90%;
-    
-}
-
-.head-text{
-    width: 375px;
-    top: 114px;
-    position: absolute;
-    left: 24px;
-}
-.head-text h1{
-    font-size: 40px;
-    font-weight: 700;
-    color: #172B4D;
-    line-height: 52px;
-}
-
-.head-text p{
-    font-size: 16px;
-    color: #344563;
-    line-height: 24px;
-}
-
```

Comment:
### [Optional Changes]
- Kindly add font-weight to this container so it can properly follow the figma design. It is important you implement this to fully follow the project requirement.